nym-vpn-app/src-tauri/src/commands/updater.rs

```diff
@@ -62,12 +63,12 @@ pub async fn install_update(
     on_event: Channel<DownloadUpdateEvent>,
 ) -> Result<(), BackendError> {
     if !*UPDATER_ENABLED {
-        error!("updater is disabled for this build");
+        info!("updater is disabled for this build");
         return Err(BackendError::internal("updater is disabled", None));
```

Comment:
We want this to be logged as error, as it is an important one in this context, we would want to have it reported in sentry too